capi/geos_c.h.in

```diff
@@ -2005,6 +2026,7 @@ extern void GEOS_DLL GEOSFree_r(
 * This function does not have a reentrant variant and is
 * available if `GEOS_USE_ONLY_R_API` is defined.
 * \return version string
+* \since 2.2
```

Comment:
Any reason all these since annotations are in this PR?

Reply:
oops?